libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -948,22 +948,22 @@ describe('expression parser functional test', () => {
         // normal case, note, we doesn't append a " yet
         let exp = Expression.parse('a[f].b[n].z');
         let path = undefined;
-        ({path, left, error} = FunctionUtils.tryAccumulatePath(exp, memory, undefined));
```

Comment:
`left` and `error` are declared globally here, but unused, so I just deleted them.